package.json

```diff
@@ -29,16 +29,16 @@
     "node": ">=10"
   },
   "dependencies": {
-    "csvtojson": "2.0.8",
-    "decimal.js": "10.1.1"
+    "csvtojson": "2.0.10",
```

Comment:
Looking at the node version in `.nvmrc`, it's fairly old (10.15.3) so probably a whole bunch of version updates are needed, but would prefer to do that separate from the data update. Would it be possible to remove the dependency update commits?

Reply:
Reverted. I'm getting a warning (since I'm using Node 8 right now) in the console about old Node. Is there a reason you're forcing 10? I don't see any issues popping up running 8.

Reply:
For the historical data, there is a gap for 2019 (before my PR). Not sure how important this archive of data is to you, but it could be added. Does the government not store this?

Reply:
Yeah there will be a gap because I didn't update for Q4 2019, so it's fine. At some point will have to figure out a less labour-intensive maintenance process. 

As for history archive, just noticed, the [Open Government Canada](https://open.canada.ca/data/en/dataset/dfa4daf1-669e-4514-82cd-982f27707ed0) site does have an RSS feed, although I tried to click `View this version` on some of them but getting an error page. Ultimately its probably beyond the scope of this repo to maintain a full history, or if so, probably better in an S3 bucket or something like that, but will leave as-is for now.

Reply:
For node version, not using anything fancy so probably would work for older versions. Usually when starting a project, I check for latest LTS on [Node](https://nodejs.org/en/), set that in `.nvmrc` in project root. Used together with [nvm shell integration](https://github.com/nvm-sh/nvm#deeper-shell-integration), as I switch around between different projects, it automatically switches to the right node version.